src/hotspot/cpu/aarch64/macroAssembler_aarch64.hpp

```diff
@@ -150,6 +150,7 @@ class MacroAssembler: public Assembler {
   void bind(Label& L) {
     Assembler::bind(L);
     code()->clear_last_insn();
+    code()->set_last_label(pc());
```

Comment:
OK, so we have added `_last_label` to shared code in `codeBuffer`, but only update it in aarch64. This would be surprising for other platforms. On the other hand, this is what we already do with `_last_insn` -- only implementing it for specific platforms. Probably fine, but it would be nice to strengthen this with asserts, maybe in separate PR.

Reply:
It reminds me it could be applied to riscv. It also need merge membar. I will move this part to a new PR.

Reply:
I need _last_label in this patch. I need it to check previous 2 instructions and they are not cross block boundary. I will create a new PR for RISCV only.